### ISSUE TITLE
[feature request] add support for customized user account and password

### ISSUE THREAD

**Issue by hsinhoyeh** [FootprintAI/multikf#7]
the default installation provides a default user (user@example.com) and password( 1234^2), but it is not safe in terms of security concerns.
this tool should be able to provide customized account information, and the use case would like 

```
./multikf add node1 --email=hello@example.com --password=mysecret
```

and multikf use this information to provision kubeflow instances.

**Comment by hsinhoyeh** [comment]
**for those who are interested to help this project, here are some information that you might need**

if you look at [manifests](https://raw.githubusercontent.com/FootprintAI/multikf/main/kfmanifests/kubeflow-manifest-v1.4.1.yaml), you are able to find 

```
staticPasswords:
    - email: user@example.com
      hash: $2y$12$4K/VkmDd1q1Orb3xAt82zu8gk7Ad6ReFR4LCP9UeYE90NLiN9Df72
```
says it uses a static password for account user@example.com and hash as its salt password.
for more information on salt passwords, you could look at this [playground](https://go.dev/play/p/DOiX2tg2hYr)
basically, it encrypted your original password and provides a hash string. and you can only use `CheckPasswordHash` function to check whether the input password is valid or not (that is what dex in Kubeflow did).

so in order to add this feature, we should make this manifests to be able to take Golang templates, so we are able to populate salt password (or email) according to the configuration specified during add node process.

For more information on Golang template, you could check [here](https://pkg.go.dev/text/template)
